src/atldld/utils.py

```diff
@@ -95,60 +96,44 @@ def get_image(image_id, folder=None, expression=False, downsample=0):
     Raises
     ------
     ValueError
-        If request to download the image failed.
+        If the image has a wrong format (determined by the dtype).
 
     References
     ----------
     [1] `AllenSDK API: ImageDownloadApi <https://allensdk.readthedocs.io/
     en/latest/allensdk.api.queries.image_download_api.html#allensdk.api.
     queries.image_download_api.ImageDownloadApi>`_
-
     """
     folder = folder or CACHE_FOLDER
-    folder = str(
-        folder
-    )  # this should guarantee that also LocalPath works (pytest uses it)
-
-    # Check the folder exists
-    if not os.path.exists(folder):
-        os.makedirs(folder)
-
-    # Create full path
-    additional_specifier = "_expression" if expression else ""
-    additional_specifier += f"_{downsample}"
-    path = "{}{}{}.jpg".format(folder, image_id, additional_specifier)
-
-    # Check image exists
-    if os.path.exists(path):
-        img = plt.imread(path)
```

Comment:
BTW `matplotlib` would very often raise a warning for big images because of some potential security risk. Do you think it is something we can completely ignore?

Reply:
I guess you're talking about the `DecompressionBombWarning` and `DecompressionBombError`? It actually comes from `Pillow` that `matplotlib.imread` uses, it's mentioned in the [docs for `PIL.Image.open`](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.open).

If I remember correctly @EmilieDel handled used to handle this before in a different repo by setting `PIL.Image.MAX_IMAGE_PIXELS` to a very high value. Actually according to the docs setting it to `None` will disable the warnings completely. I'm not a big fan of permanently changing this global value.

I think we can disable this warning locally by either setting `PIL.Image.MAX_IMAGE_PIXELS = None` right before reading the image and the setting it back to the original value straigth after, or by using `warnings.catch_warnings` together with `warnings.filterwarnings`.

I was actually trying to find an example where this warning is triggered, but couldn't find any, even at the highest resolution. Do you guys maybe have one?

Reply:
Lol, I am actually struggling to replicate the error too:D. Feel free to resolve this issue unless you want to take some further actions.

Reply:
Yes, I had this warning/error when downloading sagittal genes if I remember correctly. In `atlas_annotation`, we change the `MAX_IMAGE_PIXELS` value to the highest one expected for the given dataset to download.

Reply:
Found an example: image ID `102167293`

Reply:
So given we have a real use case, I implemented the warning ignore in f19020c.

The corresponding tests are slightly slow though. Any suggestions what to do about it?